core/domain/html_cleaner.py

```diff
@@ -146,8 +146,55 @@ def get_rte_components(html_string):
     return components
 
 
+def escape_html(html_data):
+    """This functions converts html to escaped format.
+
+    Args:
+        html_data: str. HTML string to be escaped.
+
+    Returns:
+        str. Escaped HTML string
+    """
+    replace_list = [
```

Comment:
Make this a global constant. Don't duplicate it between both functions.

Reply:
Right. Done!